CHANGELOG.md

```diff
@@ -1,5 +1,125 @@
 # Changelog
 
+## [0.59]
+
+Welcome to release 0.59 of React Native! For highlights of this release, please view the dedicated [blog post](#todo). Thanks to those who gave feedback on during the [release candidate phase]([https://github.com/react-native-community/react-native-releases/issues/41](https://github.com/react-native-community/react-native-releases/issues/79)). If you're interested in helping evaluate our next release (0.60), subscribe to the dedicated issue [here](https://github.com/react-native-community/react-native-releases/issues/#todo).
+
+### Added
+
+- Add a Metro configuration to the template with inline require/import options ([ae11993](https://github.com/facebook/react-native/commit/ae11993) by [@cpojer](https://github.com/cpojer))
+- Add Platform module for "web" target; this is now a valid third option for `Platform.OS` ([daa79b0](https://github.com/facebook/react-native/commit/daa79b0) by [@EvanBacon](https://github.com/EvanBacon))
```

Comment:
I feel like this commit message may make people thing React Native web is officially supported but that is not what this is for. Is this even worth mentioning here? I’m not sure it actually impacts anyone vs being an implementation detail for React Native web. @necolas @evanbacon, thoughts?

Reply:
It looks like that file was added only to temporarily unblock Expo to hack around an issue with requiring RN internals that themselves require Platform https://github.com/facebook/react-native/pull/23387. This actually means Expo web builds end up with 2 Platform modules and requires RN to be installed. I wouldn't mentioned this in the changelog